src/app/components/ResourceHints/index.jsx

```diff
@@ -1,24 +1,32 @@
-import React from 'react';
-import { arrayOf, string } from 'prop-types';
+import React, { Fragment } from 'react';
+import { any, objectOf, string } from 'prop-types';
 
-const ResourceHints = ({ assetOrigins }) => (
-  <>
-    {assetOrigins.map(origin => (
-      <link
-        rel="preconnect"
-        key={origin}
-        href={origin}
-        crossOrigin="anonymous"
-      />
-    ))}
-    {assetOrigins.map(origin => (
-      <link rel="dns-prefetch" key={origin} href={origin} />
-    ))}
-  </>
-);
+const ResourceHints = ({ assetOrigins, service }) => {
+  // eslint-disable-next-line global-require,import/no-dynamic-require
+  const serviceConfig = require(`../../lib/config/services/${service}`);
+  const { fonts } = serviceConfig.service.default;
+  const { fontsOrigins, ...rest } = assetOrigins;
+  let origins = Object.values(rest).flat();
+
+  if (fonts && fonts.length > 0) {
+    origins = origins.concat(fontsOrigins);
+  }
```

Comment:
Was there a reason for adding the logic to check font config and manipulate the list of origins here rather than in `getAssetOrigins`? I'd 
imagine doing it in the utility would avoid the need for destructuring/flattening the changed assetOrigins structure, as well as keeping the `ResourceHints` component reasonably free of business rules.

Reply:
I concur, following Drew's suggestion (https://github.com/bbc/simorgh/pull/4202#discussion_r334436636) we can import serviceConfigs from testHelpers and use that directly in .`.utilities/getAssetOrigins/index.js`